tests/compile_tests/audio_mem_test/newsound_test.bas

```diff
@@ -0,0 +1,14 @@
+$Console:Only
```

Comment:
If you feel like it, a few error tests wouldn't hurt. Ex. Passing negatives, or zero.

Reply:
Done.